examples/client_metadata/sd_jwt_vc_verifier_metadata.json

```
@@ -0,0 +1,8 @@
+{
+  "vp_formats": {
+    "vc+sd-jwt": {
+      "sd-jwt_alg_values": ["ES256", "ES384"],
+      "kb-jwt_alg_values": ["ES256", "ES384"]
```

I think we should keep using `alg` parameter as in other credential formats, so probably something like this:
```suggestion
  "vp_formats": {
    "vc+sd-jwt": {
      "alg": ["ES256", "ES384"]
      },
    "kb+jwt": {
      "alg": ["ES256", "ES384"]
```

---

> `alg` parameter as in other credential format

I took a deeper look and it looks like despite section 9.1 saying `For specific values that can be used, see [Appendix A](https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#alternative_credential_formats).`, appendix A does not define values from client metadata `vp_formats` property. but at the same time, [example in section 5.7](https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-5.7-10) contains `alg` claim. we should probably need to decide what approach we are taking across different credential formats.

---

We currently have this text in VP:

> vp_formats_supported: REQUIRED. An object containing a list of name/value pairs, where the name is a string identifying a Credential format supported by the Wallet. Valid Credential format identifier values are defined in Appendix A of [[OpenID.VCI](https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#OpenID.VCI)].

So I think the names under `vp_formats` are required to be Credential format identifiers. "kb+jwt" is not a Credential format identifier (nor should it be) so I don't think @Sakurann's suggestion can be used.

I think my temptation would be to merge this PR as it is, and open a new issue to discuss both this problem and the existing problem other appendix A not defining vp_formats properties.

---

I am ok externalizing my comment as an issue and merging this PR as-is

---

Should we at least change it to this then before moving. 

```
"vp_formats": {
    "vc+sd-jwt": {
      "alg": ["ES256", "ES384"]
    }
}
```

---

Not sure if this helps but I believe you could also do the following to be more explicit:

```
"vp_formats": {
    "vc+sd-jwt": {
      "kb-jwt": { "alg": [ "ES256", "ES384" ] },
      "issuer-signed-jwt": { "alg": ["ES256", "ES384"] }
    }
}
```

---

Discussed on today's WG Call - we agreed to keep the text as Jan currently has it, which is the same as it was in SD JWT VC spec.

---

@jogu do you remember the reason this direction was chosen?

---

Errr, I can't entirely remember, but I believe some combination of:

1. There was no agreement on an idea that was obviously better
2. Without any good suggestion keeping it consistent with HAIP made sense

I think it'd be fine to open an issue if you have a better suggestion, but it doesn't seem worth holding up this PR and an issue allows a more focussed discussion.